services/QuillLMS/client/app/bundles/Evidence/helpers/containerRenderHelpers.tsx

```diff
@@ -99,42 +99,34 @@ export const renderPromptSteps = ({
   completedSteps,
   doneHighlighting,
   showReadTheDirectionsModal,
-  stepsHash,
   reportAProblem,
 }) => {
   const { currentActivity, } = activities
   const { submittedResponses, hasReceivedData, } = session
+
   if (!currentActivity || !hasReceivedData) return
 
-  // sort by conjunctions in alphabetical order: because, but, so
-  const steps =  orderedSteps(activities).map((prompt, i) => {
-    // using i + 2 because the READ_PASSAGE_STEP is 1, so the first item in the set of prompts will always be 2
-    const stepNumber = i + 2
-    const canBeClicked = completedSteps.includes(stepNumber - 1) || completedSteps.includes(stepNumber) // can click on completed steps or the one after the last completed
+  const stepNumber = activeStep - 2;
```

Comment:
I know we do this all over but it might be worth a comment regarding the `-2` here.